projects/packages/videopress/src/class-initializer.php

```diff
@@ -83,6 +83,8 @@ private static function unconditional_initialization() {
 			require_once __DIR__ . '/utility-functions.php';
 		}
 
+		require_once __DIR__ . '/class-block-replacement.php';
```

Comment:
Perhaps this should go directly in the `! is_admin()` check, since that's the only place it's used?